src/edie/api.py

```diff
@@ -21,7 +22,7 @@ def __get_header(self):
     def dictionaries(self):
         headers = self.__get_header()
         return requests.get(self.endpoint + "dictionaries",
-                            headers=headers).json()
+                            headers=headers).json()["dictionaries"]
```

Comment:
This is okay as this is the only field returned by the endpoint, but it could lead to problems if the API changes and adds new fields.

Reply:
yep you're right, just didn't want to deal with dictionary keys in the main application code, but this is not the right place to handle it.